pkg/Microsoft.Private.CoreFx.NETCoreApp/netcoreapp.rids.props

```diff
@@ -1,9 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup Condition="'$(PortableBuild)' == 'true'">
```

Comment:
Consider adding a comment that for Portable builds we omit the version of the OS and treat all linux distros as linux.